java/IotSub.java

```diff
@@ -0,0 +1,6 @@
+Channel channel = ably.channels.get("device:rocket:1");
+channel.subscribe((Channel.MessageListener) message -> {
+	if (message.name == "action") {
```

Comment:
Why instead of just using `channel.subscribe("action", (Channel.MessageListener) message -> { ... })` you're checking the name in that `if`? :thinking: